src/main/java/com/sonyericsson/jenkins/plugins/bfa/GerritMessageProviderExtension.java

```diff
@@ -98,5 +98,12 @@ private void addFailureCausesFromData(StringBuilder message, FailureCauseDisplay
             .append(displayData.getLinks().getBuildUrl())
             .append(" )");
         }
+
+        if (displayData.getDownstreamFailureCauses().isEmpty() && displayData.getFoundFailureCauses().isEmpty()) {
+            message.append("Unknown (")
```

Comment:
Well I'm mostly concerned that unfamiliar users won't know/understand what that "Unknown" comes from, but if it is the same text that appears on the build page it would be easier to understand.